db/db.go

```diff
@@ -211,6 +236,11 @@ func (db *db) initialize(ctx context.Context) error {
 			return err
 		}
 
+		//err = db.Acp.Reload(ctx)
```

Comment:
todo: Please remove the commented out code

Reply:
Ah Cheers I missed this one

- [ ] Remove dead code